benchmarks/TEST_RESOURCES_HASH.txt

```diff
@@ -0,0 +1 @@
+81cbf23054ce82abd06d511f04dd57b47a02b120552f3a3c770f3e78e8b6be9e
```

Comment:
yeah, we eventually want anyone to be able to run the benchmarks, and they need this to download the test data from the Quilt bucket (although they can't yet because the bucket still needs to be made public)